components/TopBarProfileMenu.js

```diff
@@ -219,7 +219,10 @@ class TopBarProfileMenu extends React.Component {
     const main = document.querySelector('main');
     main.addEventListener('keydown', this.handleKeyPress);
     main.addEventListener('click', this.onClickOutside);
-    if (!getFromLocalStorage(LOCAL_STORAGE_KEYS.ACCESS_TOKEN)) {
+    if (
+      !getFromLocalStorage(LOCAL_STORAGE_KEYS.ACCESS_TOKEN) ||
+      !getFromLocalStorage(LOCAL_STORAGE_KEYS.TWO_FACTOR_AUTH_TOKEN)
```

Comment:
I don't get this check: why should we consider that user's not loading if there's no 2FA token in the local storage (which will be the case most of the time)? I would have understood a `&&`, but not a `||`.